storage_service/storage_service/urls.py

```diff
@@ -125,5 +125,11 @@ def startup():
             utils.set_setting(loc_info['default_setting'], [new_loc.uuid])
             LOGGER.info('Set %s as %s', new_loc, loc_info['default_setting'])
 
+    # Set any interrupted package moves to failed
+    packages = locations_models.Package.objects.filter(status=locations_models.Package.MOVING)
+    for package in packages:
+        package.status = locations_models.Package.MOVE_FAILED
+        package.save()
```

Comment:
This can be done in one line (and one query), using [`.update()`](https://docs.djangoproject.com/en/dev/ref/models/querysets/#update)  There's a lot of useful things on the queryset reference page, check it out.